include/nnstreamer-edge-event.h

```diff
@@ -30,6 +30,7 @@ typedef enum {
   NNS_EDGE_EVENT_NEW_DATA_RECEIVED,
   NNS_EDGE_EVENT_CALLBACK_RELEASED,
   NNS_EDGE_EVENT_CONNECTION_CLOSED,
+  NNS_EDGE_EVENT_CONNECTION_COMPLETED,
```

Comment:
Does this message indicate that the network connection was successfully (completed)established? Or does it mean that the network connection is active?

Reply:
It means  that the network connection was successfully established.
If network connection status is changed to inactive, `NNS_EDGE_EVENT_CONNECTION_CLOSED` event will be invoked.

Reply:
Thank you for the detailed explanation. :pray: